src/utils/limits.py

```diff
@@ -0,0 +1,63 @@
+# limits.py
+# author: Daniel Jauergui
+# date: 3-31-2015
+
+
+class Limits(object):
+    def axis_missing(self, board, x, axis):
+        """Verify the missing axis using the board
```

Comment:
Again the method documentation should explain what it does, describe the parameters in the context, if we are not adding these and only documenting the values types we are not gaining anything with this documentation